src/common/components/table-pk-picker/table-pk-picker.component.module.css

```diff
@@ -41,6 +41,7 @@
 /* Active */
 .select-select:hover {
   --border-select-color: var(--input-border-color-active);
+  transform: translateY(-50%) scale(1.2);
```

Comment:
Remove this change, it is not necessary to change anything in this file. :)

Reply:
It's fixed ;)